WTG.Analyzers/Analyzers/Whitespace/WhitespaceAnalyzer.cs

```diff
@@ -147,5 +147,13 @@ static string LeadingWhitespace(string text)
 		// Must consist of one or more tab followed by up to 3 spaces.
 		// (sometimes visual studio likes to add a few spaces to spaces to align with something on the previous line.)
 		static readonly Regex acceptableLeadingWhitespace = new Regex(@"^\t*[ ]{0,3}$", RegexOptions.ExplicitCapture);
+
+		static string HumanReadablePlatformNewLine { get; } = Environment.NewLine switch
```

Comment:
not sure why this needs to be a property since it's private :)

```csharp
static readonly string HumanReadablePlatformNewLine = Environment.NewLine switch ...
```

Reply:
Habit, I guess.